kelsvilla/LegoMindstormsVSExtension
# Demo Webpage

Adding and changing existing HTML files for our demo page on ghpages.

### Testing
1. Install the Live Server extension on VSCode if you don't have it already
    - ![image](https://github.com/kelsvilla/LegoMindstormsVSExtension/assets/89543725/e056ab0d-8e5c-4a98-b4c5-2a55548a87e1)

2. Click on "Go Live" at the bottom right corner of VSCode
    - ![image](https://github.com/kelsvilla/LegoMindstormsVSExtension/assets/89543725/43a17a52-485d-4905-811e-2d176e046303)

This should open localhost page of our demo page on a browser. Still a work in progress, but will be updating the photos at [Trello](https://trello.com/c/7mDc3Jnk/5-add-demo-webpage)
